pkg/jobrunaggregator/jobrunaggregatorapi/types_row_backenddisruption.go

```diff
@@ -20,5 +22,173 @@ SELECT
 FROM openshift-ci-data-analysis.ci_data.BackendDisruption
 INNER JOIN openshift-ci-data-analysis.ci_data.BackendDisruption_JobRuns as JobRuns on BackendDisruption.JobRunName = JobRuns.Name
 INNER JOIN openshift-ci-data-analysis.ci_data.Jobs on JobRuns.JobName = Jobs.JobName
+`
+
+	unifiedBackendDisruptionBackendsSchema = `
+SELECT 
+  JobRuns.name as JobRunName,
+  Jobs.Jobname as JobName,
+  JobRuns.StartTime as JobRunStartTime,
+  JobRuns.ReleaseTag as ReleaseTag,
+  BackendDisruption.BackendName as BackendName,
+  BackendDisruption.DisruptionSeconds as DisruptionSeconds,
+  JobRuns.Cluster as Cluster,
+  Jobs.Platform as Platform,
+  Jobs.Network as Network,
+  Jobs.IPMode as IPMode,
+  Jobs.Topology as Topology,
+  Jobs.Release as Release,
+  Jobs.FromRelease as FromRelease,
+  if(Jobs.FromRelease="",false,true) as IsUpgrade,
+FROM openshift-ci-data-analysis.ci_data.BackendDisruption
+INNER JOIN openshift-ci-data-analysis.ci_data.BackendDisruption_JobRuns as JobRuns on JobRuns.Name = BackendDisruption.JobRunName
+INNER JOIN openshift-ci-data-analysis.ci_data.Jobs on JobRuns.JobName = Jobs.JobName
+`
+
+	unifiedBackendDisruptionBackendByCountSchema = `
+SELECT 
+  Jobs.Jobname as JobName,
+  ByCount.BackendName as BackendName,
+  ByCount.DisruptionSeconds as DisruptionSeconds,
+  ByCount.NumberOfRuns as NumberOfRuns,
+  Jobs.Platform as Platform,
+  Jobs.Network as Network,
+  Jobs.IPMode as IPMode,
+  Jobs.Topology as Topology,
+  Jobs.Release as Release,
+  Jobs.FromRelease as FromRelease,
+  if(Jobs.FromRelease="",false,true) as IsUpgrade,
+FROM openshift-ci-data-analysis.ci_data.BackendDisruption_ByCountByJob as ByCount
+INNER JOIN openshift-ci-data-analysis.ci_data.Jobs on ByCount.JobName = Jobs.JobName
+`
+
+	unifiedBackendDisruptionByCountByJobSchema = `
```

Comment:
I'm assuming this is supposed to match `BackendDisruption_ByCountByJob`.
But it doesn't match.

Big Query shows:

```
SELECT
    JobName, 
    BackendName, 
    DisruptionSeconds,
    count(JobRunName) as NumberOfRuns
from 
    `openshift-ci-data-analysis.ci_data.BackendDisruption_Unified_Backends`
WHERE
    JobRunStartTime BETWEEN TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 4 DAY)
        AND
        TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 0 DAY)
group by 
    JobName, BackendName, DisruptionSeconds```